docs/snp.md

```diff
@@ -56,6 +56,10 @@ Setup the host by building SNP patched versions of qemu, ovmf and the linux kern
 ```
 ./snp.sh setup-host
 ```
+Coconut-SVSM: To set up the host by building IGVM, SVSM, QEMU, OVMF, and the Linux kernel, use the following command:
+```
+./snp.sh --svsm setup-host
+```
```

Comment:
```suggestion
The `--svsm` option can be specified with the above command to set-up the host for SVSM. This will build a Coconut-SVSM kernel as well as the IGVM, SVSM, OVMF and QEMU launch dependencies.
```
```

Reply:
You are asking remove command and add verbal explanation. I would like to keep command as well in documentation. This gives clear instruction on where they should add "--svsm" option. @ryansavino, any suggestions?